extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "commercetools-adyen-integration-extension",
-  "version": "9.0.0",
+  "version": "9.0.2",
```

Comment:
You might wanna go higher to 9.0.3 now.

Reply:
Yes I tested, 
https://github.com/commercetools/commercetools-adyen-integration/pull/803/commits/746cdaacf67afb086fc23669742c7952a6214302